### ISSUE TITLE
What's the "Shadow Content"?

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#140]
Just curious about something called the "Shadow Content" that I see when I inspect the PDOM?

What is it, and is it needed? 
I am curious if it has any effect on the Accessibility Tree?

Assigning to @jessegreenberg, but @zepumph, feel free to comment if you know about it.

**Comment by jessegreenberg** [comment]
Hmm, we aren't doing anything special with the shadow DOM so I am honestly not sure. Perhaps it is just native shadow DOM stuff that comes with native elements? I am honestly not sure @terracoda,  can you past a screenshot of what you are seeing?

**Comment by terracoda** [comment]
Yes, a screenshot. I forgot to post that. This is what I see with Safari's inspector.
<img width="549" alt="Screen Shot 2019-06-03 at 6 28 45 PM" src="https://user-images.githubusercontent.com/1902116/58937458-c73a0280-8772-11e9-84db-ac9dd73da1c5.png">

To me the chunk of code just looks like empty `div`s, but it is not clear to me that they are really there. The grey box around the block is different from the surrounding code.

**Comment by jessegreenberg** [comment]
Thanks for the screenshot! OK, it looks like just the browser's native shadow DOM for the slider. Here is some info I found about the shadow DOM for sliders in particular:
https://glazkov.com/2011/01/14/what-the-heck-is-shadow-dom/

In order for it to show up for me in Chrome I had to check "Show user agent Shadow DOM" in settings, there may be a way to disable it in Safari if it is bothersome @terracoda.
